web/modules/custom/react_tools/tools/apps/react-entitlements-page/src/App.js

```diff
@@ -6,12 +6,11 @@ import Table from './components/table'
 import * as axios from 'axios';
 import * as _ from 'lodash';
 
-
 class App extends Component {
   constructor() {
     super();
 
-    this.baseUrl = process.env.NODE_ENV === 'development' ? 'http://movemil.local/' : '/';
+    this.baseUrl = process.env.NODE_ENV === 'development' ? 'https://stage.move.mil/' : '/';
```

Comment:
@Jsarnik do you think we should commit this change? Or did you forget to remove it?

Reply:
ugh, yes I forgot to remove it. I will create a new PR when I fix this.